frontend/app/src/composables/filters/events.ts

```diff
@@ -33,7 +33,10 @@ export type Matcher = SearchMatcher<
 >;
 export type Filters = MatchedKeyword<HistoryEventFilterValueKeys>;
 
-export const useHistoryEventFilter = (disableProtocols: boolean) => {
+export const useHistoryEventFilter = (
+  disableProtocols: boolean,
+  disableLocations = false
```

Comment:
```suggestion
  disabled: { protocols?: boolean, locations?: boolean } 
```

Can we maybe change the syntax to this? 

The check should look like `if(!disabled?.protocols)` etc